frontend/src/components/ui/FeatureCTABanner.tsx

```diff
@@ -0,0 +1,110 @@
+"use client";
+import React, { useState, useEffect } from 'react';
+import { Box, Button, Paper, Typography, IconButton, Container, useTheme } from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import Link from 'next/link';
+
+interface FeatureCTABannerProps {
+  onClose?: () => void;
+  actionUrl?: string;
+  buttonText?: string;
+}
+
+export default function FeatureCTABanner({
+  onClose,
+  actionUrl,
+  buttonText = "Sign Up Now"
+}: FeatureCTABannerProps) {
+  const [open, setOpen] = useState<boolean>(false);
+  const bannerKey = 'feature-favorites-banner-dismissed';
+  const theme = useTheme();
+  useEffect(() => {
+    // Check if the user has previously dismissed this banner
+    const isBannerDismissed = localStorage.getItem(bannerKey) === 'dismissed';
+    if (!isBannerDismissed) {
+      setOpen(true);
+    }
+  }, [bannerKey]);
+
+  const handleClose = (): void => {
+    setOpen(false);
+    // Store the dismissal in localStorage
+    localStorage.setItem(bannerKey, 'dismissed');
+    if (onClose) onClose();
+  };
+
+  if (!open) return null;
+
+  return (
+    <Paper
+      elevation={0}
+      sx={{
+        width: '100%',
+        background: theme.palette.mode === 'light' ? 'linear-gradient(90deg, #009688,rgb(74, 170, 210))' : theme.palette.background.default,
+        color: 'white',
+        py: 2,
+        position: 'relative',
+        borderRadius: 0
+      }}
+    >
+      <Container maxWidth="lg">
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'space-between',
+            flexWrap: { xs: 'wrap', md: 'nowrap' },
+            gap: 2,
+          }}
+        >
+          <Box sx={{ display: 'flex', alignItems: 'center', flexGrow: 1 }}>
+            <Box>
+              <Typography variant="subtitle1" sx={{ fontWeight: 'bold', mb: 0.5 }}>
+                <FavoriteIcon sx={{ mr: 1, fontSize: 20, verticalAlign: 'text-bottom' }} />
+                Save your favorite artists for later
+              </Typography>
+              <Typography variant="body2">
+                Create an account to save artists to your Favorites list for easy access! Coming soon: specialties like ethnicities and styles.
```

Comment:
nit: Create an account to save your favorite artists for easy access!
nit: can we put coming soon on a separate line?

Reply:
updated the wording! new line was making the banner too large IMO
<img width="1696" alt="Screenshot 2025-04-29 at 11 22 47 AM" src="https://github.com/user-attachments/assets/8d430009-0593-4ce8-8eaa-cf3960c3c66a" />